cobra/flux_analysis/sampling.py

```diff
@@ -355,10 +359,11 @@ def _sample_chain(args):
 
     center and n_samples are updated locally and forgotten afterwards.
     """
-    sampler, n = args       # has to be this way to work in Python 2.7
+    sampler, n, idx = args       # has to be this way to work in Python 2.7
     center = sampler.center
+    np.random.seed(int(time() * idx + idx) % np.iinfo(np.int32).max)
```

Comment:
so simply `np.random.seed()` doesn't do what its supposed to according to the docs?